hack/update-image-tags.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -eu -o pipefail
+
+dir=$1
+image_tag=$2
+
+find "$dir" -type f -name '*.yaml' | while read -r f ; do
+  sed "s@argoproj/\(.*\):.*@argoproj/\1:${image_tag}@" "$f" > .tmp
```

Comment:
Minor, but I think this is more clear:
```suggestion
  sed "s|argoproj/\(.*\):.*|argoproj/\1:${image_tag}|" "$f" > .tmp
```